docs/walkthrough/create-training-data.md

```diff
@@ -68,13 +84,22 @@ This is another English sentence    Dies ist ein weiterer englischer Satz
 ```
 
 ```{warning}
-When reading in files constructed in this way, a unique label is generated for each pair. Some sampling methods that require more than two elements per class will not work because each class in the dataset will only contain a single pair of elements.
-
-
+When reading in files constructed in this way, a unique label is generated for each pair.
+Some sampling methods that require more than two elements per class will not work because each class in the dataset will only contain a single pair of elements.
 ```
 
 ````
 
+It is worth noting that we support the following dialects:
```

Comment:
"We support the following dialects of CSV:"

Direct language, and you should specify dialect of what since I don't see it from context.